### PR TITLE
Nikeananthclone patch 1

### PR DESCRIPTION
**Pull request by nikeananthclone**

### CODE REVIEW

**Comment by nikeananthclone** [comment]
check in dude

File: color [hunk 1]
```diff
@@ -8,4 +8,5 @@ nike2
 nike3
 nike4
 nike5
-
+nokia1
```
Comment (nikeananth) [inline color:11]: check this part

File: color [hunk 2]
```diff
@@ -8,4 +8,5 @@ nike2
...
+nokia2
```
Comment (nikeananth) [inline color:12]: change this line.

**Review by nikeananth — CHANGES_REQUESTED** [review]
please check the code.

File: color (hunk 2 above)
Comment (nikeananthclone) [reply color:12]: added

**Pull request merged**